test/parallel/test-net-connect-immediate-destroy.js

```diff
@@ -2,7 +2,11 @@
 const common = require('../common');
 const net = require('net');
 
-const socket = net.connect(common.PORT, common.localhostIPv4,
-                           common.mustNotCall());
-socket.on('error', common.mustNotCall());
-socket.destroy();
+const s = net.createServer();
+s.listen(0);
+const port = s.address().port;
+s.close(() => {
+  const socket = net.connect(port, common.localhostIPv4, common.mustNotCall());
+  socket.on('error', common.mustNotCall());
+  socket.destroy();
+});
```

Comment:
> Creating a server just to secure a port did seem unnecessary but would it be alright if still kept it under parallel?

I think it's the only way if we want to keep it under parallel. BTW, I've just noticed there are other tests in the PR with the same issue: attempting the connection after closing the server. Could you update those too? Then I'll launch a CI job. Thanks"